bokeh/model.py

```diff
@@ -235,16 +235,16 @@ def __init__(self, **kwargs):
         super().__init__(**kwargs)
         default_theme.apply_to_model(self)
 
-    def __str__(self):
+    def __str__(self) -> str:
         return "%s(id=%r, ...)" % (self.__class__.__name__, getattr(self, "id", None))
 
     __repr__ = __str__
 
     @property
-    def id(self):
+    def id(self) -> str:
         return self._id
 
-    name = String(help="""
+    name: str = String(help="""
```

Comment:
I'm fairly sure these annotations are causing the sphinx fairly. Are they necessary to add now? Can we make a separate issue to do late and handle the necessary sphinxext updates then as well?

Reply:
Nevermind it's a trivial fix, I have pushed the change to this branch.